envwarden

```diff
@@ -48,6 +48,20 @@ while [[ $# > 0 ]]; do
     shift
 done
 
+which jq > /dev/null 2>&1
```

Comment:
This is nice, but not really necessary. If you call `jq` and it doesn't exist, you will get a `jq: command not found` anyway.

Reply:
True, but getting version will fail too.